requirements.txt

```diff
@@ -37,3 +37,6 @@ git+https://github.com/dPys/deepbrain.git@master
 urllib3>=1.25.4
 mplcyberpunk>=0.1.11
 numba>=0.51.2
+anytree==2.8.0
+gensim==3.8.3
+graspy==0.3.0
```

Comment:
So graspy has been repackaged as graspologic, and we can just go with that.

Reply:
Ok, great! I got a little confused because the pipeline failed when I first made the pull request and the error was that the module was missing, so I went ahead and added it despite not personally using it myself. I think at one point when I updated my local repo with the changes made in the upstream I had some merge conflicts, so I bet I accidently chose to keep the outdated package.